pkg/tfbridge/rawstate.go

```diff
@@ -493,8 +493,20 @@ func RawStateComputeDelta(
 	pv := resource.NewObjectProperty(outMap)
 
 	delta := ih.delta(pv, v)
-	vWithoutTimeouts := v.Remove("timeouts")
-	err := delta.turnaroundCheck(ctx, newRawStateFromValue(schemaType, vWithoutTimeouts), pv)
+	var rawState rawstate.RawState
+	if _, hasTimeouts := outMap["timeouts"]; hasTimeouts {
```

Comment:
Nit: curious if we could use schema to check for "timeouts" being part of schema rather than guessing from the outMap? `schemaMap` is the "static" source of information.